requirements.txt

```diff
@@ -1,5 +1,4 @@
+incremental
 twisted
 pyOpenSSL
 service_identity
-systemd
-txdbus
```

Comment:
These changes might break the `virtualenv` setup on linux for `make check`. But mac support is more important as far as I'm concerned, so let's just merge this.

Reply:
I know. And I had a couple of solutions in mind for it. - - One is to have a separate requirements file, but I didn't like it since the name here is pretty standard.

- Second is to have a `configure` script (autotools style). I like this solution but it will take more time implementing and testing (since it will break the packaging for linux).

I'll start later on the second solution. What do you think?